requirements/edx/github.txt

```diff
@@ -78,7 +78,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.4#egg=XBlock==0.4.4
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@0.2.6#egg=ora2==0.2.6
+-e git+https://github.com/edx/edx-ora2.git@0.2.7#egg=ora2==0.2.7
```

Comment:
The -e means, install as an editable working tree.  We never needed it, it's been cargo-culted from the beginning.  It's allowed us to have bugs in our setup.py files that we don't find until much later.

The instructions at the top of the file were an attempt to get people to stop using it. :)